spacetemplate/assets/scrum.yaml

```diff
@@ -5,7 +5,7 @@ space_template:
   description: >
     An iterative and incremental agile software development framework for
     managing product development.
-  can_construct: yes
+  can_construct: no
```

Comment:
Is this intentional?

Reply:
Yes. The legacy Scrum template is to be deprecated.

Reply:
@michaelkleinhenz why is that?